packages/blocks/src/api/factory.js

```diff
@@ -69,14 +67,18 @@ export function createBlock( name, blockAttributes = {}, innerBlocks = [] ) {
 			} );
 
 			// Ensure value passed is always an array, which we're expecting in
-			// the RichText component to handle the deprecated value.
-			if ( typeof result[ key ] === 'string' ) {
-				result[ key ] = [ result[ key ] ];
-			} else if ( ! Array.isArray( result[ key ] ) ) {
-				result[ key ] = [];
+			// the RichText getColorObjectByAttributeValuescomponent to handle the deprecated value.
```

Comment:
Is this copy and paste issue `getColorObjectByAttributeValuescomponent`?

Reply:
hahaha :) sorry about that